skills/lark-drive/references/scripts/publish_gate.py

```
@@ -0,0 +1,338 @@
+    """Shared gates: sensitive and unresolved-conflict material never ships."""
+    sensitivity = str(item.get("sensitivity") or "").strip().lower()
+    if sensitivity == "prohibited":
+        hard_reasons.append("敏感等级 prohibited，禁止入库")
+    elif sensitivity == "restricted":
+        review = str(item.get("sensitive_review_status") or "").strip().lower()
+        if review != "approved":
+            hard_reasons.append("受限敏感内容未通过审核（sensitive_review_status 非 approved）")
+
+    conflict = str(item.get("conflict_status") or "").strip().lower()
+    if conflict in BLOCKING_CONFLICTS:
+        hard_reasons.append(f"存在未裁决冲突（conflict_status={conflict}）")
```

Ruff (0.16.3)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)